File: C#/LINQtoDB/DataConnectoins/DefaultDataConnection.cs
using LinqToDB;
using LinqToDB.Data;
using Models;

namespace DataConnections;

public class DefaultDataConnection : DataConnection
{
    public DefaultDataConnection() : base("Default") {
        this.CreateTable<Department>(tableOptions: TableOptions.CreateIfNotExists);
        this.CreateTable<Person>(tableOptions: TableOptions.CreateIfNotExists);
        this.CreateTable<Employee>(tableOptions: TableOptions.CreateIfNotExists);
    }

    public ITable<Department> Departments => this.GetTable<Department>();
    public ITable<Person> People => this.GetTable<Person>();
    public ITable<Employee> Employees => this.GetTable<Employee>();
}

File: FSharp/LINQtoDB/DataConnections/DefaultDataConnection.fs
namespace DataConnections

open LinqToDB
open LinqToDB.Data
open Models

type DefaultDataConnection (configurationString: string) =
  inherit DataConnection (configurationString)

  new () as this =
    new DefaultDataConnection ("Default")
    then
      this.CreateTable<Department> (tableOptions=TableOptions.CreateIfNotExists) |> ignore
      this.CreateTable<Person> (tableOptions=TableOptions.CreateIfNotExists) |> ignore
      this.CreateTable<Employee> (tableOptions=TableOptions.CreateIfNotExists) |> ignore

  member this.Departments = this.GetTable<Department> ()
  member this.People = this.GetTable<Person> ()
  member this.Employees = this.GetTable<Employee> ()
